elements/lisk-transactions/src/fee.ts

```diff
@@ -22,9 +22,10 @@ interface BaseFee {
 }
 
 interface Options {
-	readonly minFeePerByte: number;
-	readonly baseFees: BaseFee[];
-	readonly numberOfSignatures: number;
+	readonly minFeePerByte?: number;
+	readonly baseFees?: BaseFee[];
+	readonly numberOfSignatures?: number;
+	readonly numberOfEmptySignatures?: number;
```

Comment:
do we need this extra param? from signatures array we can know the empty buffer and take the count?

Reply:
we don't use `signatures` param in this function, so we need to know how many is required and how many signatures are empty